.github/workflows/release-changelog.yml

```diff
@@ -0,0 +1,49 @@
+name: Changelog vs Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Why only starting with a `v` ?
To be honest, I see no point to request it

Reply:
Done